src/experimental/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './effects';
+export * from './lib';
```

Comment:
I recreated my original filestructure in the experimental directory.  effect.js and effect-manager.js were originally in src/lib/; now they are in src/experimental/lib/.  Does that seem like a sensible way of doing things?